config/config.py

```
@@ -1,292 +1,450 @@
+    'get_category_config',
+    'get_config_structure',
+    'invalidate_config_cache',
+    'validate_and_format_config_value',
+    'CONFIGURABLE_OPTIONS',
+    'RESTART_REQUIRED_KEYS',
+    'CONFIG_SECTIONS',
+    'CONFIG_CATEGORIES',
+    'CONFIG_PATH',
+    'ConfigSchema',
+    'ConfigError',
+    'ConfigLoadError',
```

Ruff

Undefined name `ConfigLoadError` in `__all__`

(F822)